timesketch/lib/analyzers/__init__.py

```diff
@@ -18,6 +18,7 @@
 from timesketch.lib.analyzers import browser_search
 from timesketch.lib.analyzers import browser_timeframe
 from timesketch.lib.analyzers import chain
+from timesketch.lib.analyzers import crash
```

Comment:
can you name this something else, since this is only for windows crash files, wondering whether better to have it `win_crash` so it is more explicit.